### PR TITLE
Updated logic to create baseline

### PR DESCRIPTION
**Pull request by IanKBovard**
Updated routing,
changed imports to use index.js file
reorganized files into folders,
renamed footer and header

### CODE REVIEW

File: src/components/Header/styles.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  @media (max-width: 960px){
+    img {
+      width: auto;
+      max-width: 98%;
+      max-height: 275px;
+    }
+  }
+`;
+
+export const BrickWall = styled(Box)`
```
Comment (britt6612) [inline src/components/Header/styles.js:22]: This can probably come out

File: src/components/Header/styles.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/* (C) Copyright 2019 Hewlett Packard Enterprise Development LP. */
```
Comment (britt6612) [inline src/components/Header/styles.js:1]: Actually this whole file is not being imported anymore in header index I think. we. can delete

Comment (IanKBovard) [reply src/components/Header/styles.js:1]: can do

File: src/components/Header/styles.js (hunk 1 above)
Comment (IanKBovard) [reply src/components/Header/styles.js:22]: can do

**Pull request merged**